lib/components/artist/related_artists/related_artist.js

```diff
@@ -52,6 +49,19 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+  prepareVariables: (prevVariables) => {
+    const {windowHeight, windowWidth} = Dimensions.get('window');
+    let newWidth = (windowWidth - 60)/2;
+    console.log(newWidth);
+    console.log('window width: ' + windowWidth);
```

Comment:
View’s have not been created/laid out yet when Relay is doing its initial query building. We could probably pass data in from outside when creating the artist component tree, though.